gocse/schema/json_schema.go

```diff
@@ -0,0 +1,61 @@
+package schema
+
+import (
+	"fmt"
+	"log"
+
+	"go.mongodb.org/mongo-driver/bson"
+)
+
+func CreateJSONSchema(dataKeyBase64 string) bson.Raw {
+	jsc := `{
+	"bsonType": "object",
+	"encryptMetadata": {
+		"keyId": [
+			{
+				"$binary": {
+					"base64": "%s",
+					"subType": "04"
+				}
+	        }
+		]
+	},
+	"properties": {
+		"insurance": {
+			"bsonType": "object",
+			"properties": {
+				"policyNumber": {
+					"encrypt": {
+						"bsonType": "int",
+						"algorithm": "AEAD_AES_256_CBC_HMAC_SHA_512-Deterministic"
+					}
+				}
+			}
+		},
+		"medicalRecords": {
+			"encrypt": {
+				"bsonType": "array",
+				"algorithm": "AEAD_AES_256_CBC_HMAC_SHA_512-Random"
+			}
+		},
+		"bloodType": {
+			"encrypt": {
+				"bsonType": "string",
+				"algorithm": "AEAD_AES_256_CBC_HMAC_SHA_512-Random"
+			}
+		},
+		"ssn": {
+			"encrypt": {
+				"bsonType": "int",
+				"algorithm": "AEAD_AES_256_CBC_HMAC_SHA_512-Deterministic"
+			}
+		}
+	}
+}`
```

Comment:
Indentation issue?

Reply:
I'm not sure what you mean. It looks correctly lined up to me? https://www.dropbox.com/s/4f93qt73qqdp37s/Screen%20Shot%202020-11-24%20at%2012.00.12%20PM.png